ironfish/src/memPool/feeEstimator.test.ts

```diff
@@ -411,7 +405,7 @@ describe('FeeEstimator', () => {
         },
       ])
 
-      expect(fee).toBe(BigInt(6))
+      expect(fee).toBe(BigInt(9))
```

Comment:
This looks right. I think I'd like for this to be less of a magic number and derive it from the fee rate and the size of the transaction, but I think that can happen separately

I think the change in the fee estimate came from different transactions/fees being added to the chain in the old fixtures

Reply:
Cool, sounds good!